apps/wing-console/console/app/demo/main.w

```diff
@@ -8,11 +8,11 @@ let errorService = new cloud.Service(inflight () => {}) as "ErrorService";
 
 let errorResource = new sim.Resource(inflight () => {
   util.sleep(5s);
-  throw "Oops";
+  //throw "Oops";
```

Comment:
For some reason, the testing sim is not working for resources that throw exceptions :) .

working on it